bot.js

```
@@ -56,7 +56,7 @@ client.on('message', msg => {
     
     
 var said = msg.content.toLowerCase(); //declare said
-    if (msg.content.startsWith(prefix + "cookpie")) {
+    if (msg.content.startsWith(prefix + "cookpie") && msg.channel.id === "382504483483025410") {
```

Let's throw an error message if the channel is not the bots-spam channel. 
`if(msg.content.startsWith(prefix + "cookpie") {
if(msg.channel.id !== "382504483483025410") {
   msg.delete(3000)
   msg.reply("Pies can only be cooked in <#BOTCHNLID>!").then(msg => msg.delete(3000))
   return; //RETURN IS VERY IMPORTANT
}
//cookpie code here
}`

---

Ok...